annotator/src/test/java/org/cbioportal/annotator/internal/GenomeNexusImplTest.java

```diff
@@ -47,415 +48,416 @@
-        map.put("SAMPLE-VARIANT-9", "Nonsense_Mutation");
-        map.put("SAMPLE-VARIANT-10", "Frame_Shift_Ins");
-        map.put("SAMPLE-VARIANT-11", "Frame_Shift_Ins");
-        map.put("SAMPLE-VARIANT-12", "Frame_Shift_Ins");
-        map.put("SAMPLE-VARIANT-13", "Frame_Shift_Del");
-        map.put("SAMPLE-VARIANT-14", "Frame_Shift_Del");
-        map.put("SAMPLE-VARIANT-15", "Frame_Shift_Del");
-        map.put("SAMPLE-VARIANT-16", "Frame_Shift_Ins");
-        map.put("SAMPLE-VARIANT-17", "Frame_Shift_Del");
-        return map;
-    }
-
-    private Map<String, String> makeMockExpectedProteinChange() {
-        Map<String, String> map = new HashMap<>();
-        map.put("SAMPLE-VARIANT-1", "p.G432delinsES");
-        map.put("SAMPLE-VARIANT-2", "p.H1034delinsPY");
-        map.put("SAMPLE-VARIANT-3", "p.Q10delinsHR");
-        map.put("SAMPLE-VARIANT-4", "p.F594_D600delinsSPPPH");
-        map.put("SAMPLE-VARIANT-5", "p.E1350del");
-        map.put("SAMPLE-VARIANT-6", "p.S124_S127del");
-        map.put("SAMPLE-VARIANT-7", "p.K416E");
-        map.put("SAMPLE-VARIANT-8", "p.X218_splice");
-        map.put("SAMPLE-VARIANT-9", "p.Q928*");
-        map.put("SAMPLE-VARIANT-10", "p.L431Vfs*22");
-        map.put("SAMPLE-VARIANT-11", "p.D1450*");
-        map.put("SAMPLE-VARIANT-12", "p.A765Rfs*98");
-        map.put("SAMPLE-VARIANT-13", "p.R646Gfs*22");
-        map.put("SAMPLE-VARIANT-14", "p.S378Ffs*6");
-        map.put("SAMPLE-VARIANT-15", "p.P692Lfs*43");
-        map.put("SAMPLE-VARIANT-16", "p.M209Ifs*2");
-        map.put("SAMPLE-VARIANT-17", "p.S378Ffs*5");
-        return map;
-    }
-
-    private Map<String, String> makeMockExpectedGenomicLocations() {
-        Map<String, String> map = new HashMap<>();
-        map.put("SAMPLE-VARIANT-1", "4,9784947,9784948,-,AGA");
-        map.put("SAMPLE-VARIANT-2", "3,14940279,14940280,-,CAT");
-        map.put("SAMPLE-VARIANT-3", "16,9057113,9057114,-,CTG");
-        map.put("SAMPLE-VARIANT-4", "13,28608258,28608275,CATATTCATATTCTCTGA,GGGGTGGGGGGG");
-        map.put("SAMPLE-VARIANT-5", "22,36689419,36689421,CCT,-");
-        map.put("SAMPLE-VARIANT-6", "3,14106026,14106037,CCAGCAGTAGCT,-");
-        map.put("SAMPLE-VARIANT-7", "22,29091840,29091841,TG,CA");
-        map.put("SAMPLE-VARIANT-8", "19,46141892,46141893,TC,AA");
-        map.put("SAMPLE-VARIANT-9", "11,62393546,62393547,GG,AA");
-        map.put("SAMPLE-VARIANT-10", "1,65325832,65325833,-,G");
-        map.put("SAMPLE-VARIANT-11", "4,77675978,77675979,-,C");
-        map.put("SAMPLE-VARIANT-12", "8,37696499,37696500,-,G");
-        map.put("SAMPLE-VARIANT-13", "10,101953779,101953779,T,-");
-        map.put("SAMPLE-VARIANT-14", "6,137519505,137519506,CT,-");
-        map.put("SAMPLE-VARIANT-15", "3,114058003,114058003,G,-");
-        map.put("SAMPLE-VARIANT-16", "9,135797242,135797242,C,AT");
-        map.put("SAMPLE-VARIANT-17", "6,137519505,137519506,CT,A");
-        return map;
-    }
+//    @Autowired
```

Comment:
This doesn't seem right ... we should not be commenting out the entire unit test class ... if we want to delete it we should delete it [but I would ask why it was being deleted].